src/routes/console/wizard/cloudOrganization/confirmDetails.svelte

```diff
@@ -123,9 +126,9 @@
         {#if $createOrganization.billingPlan !== BillingPlan.STARTER}
             <p class="text u-margin-block-start-16">
                 This amount, and any additional usage fees, will be charged on a recurring 30-day
-                billing cycle after your trial period ends on <b
-                    >{toLocaleDate(billingPayDate.toString())}</b
-                >.
+                billing cycle{TRIAL_PERIOD_OVERRIDE
+                    ? '.'
+                    : ` after your trial period ends on ${toLocaleDate(billingPayDate.toString())}`}.
```

Comment:
There seems to be an extra period:

<img width="1169" alt="image" src="https://github.com/appwrite/console/assets/1477010/5698bff0-24d9-43bb-971c-d7e983790bdb">